cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -95,22 +95,44 @@ CREATE INDEX IF NOT EXISTS census_state_geo_id_index ON demographics (state_cens
 CREATE INDEX IF NOT EXISTS census_county_geo_id_index ON demographics (county_census_geo_id);
 CREATE INDEX IF NOT EXISTS geom_index ON demographics USING GIST (geom);
 
+--- Tables related to U.S. demographic information.
+
+-- TODO(breuch): Add insert statement once the zipcode -> demographics
+
+CREATE TABLE IF NOT EXISTS aggregate_us_demographics
+(
+    census_geo_id                varchar(255) NOT NULL,
+    geo_type                     varchar(255) NOT NULL,
+    name                         varchar(255) NOT NULL,
+    average_home_age             real,
+    average_income_level         real,
+    average_social_vulnerability real,
+    black_population             real,
+    white_population             real,
+    total_population             real,
+    under_five_population        real,
+    poverty_population           real,
+    geom                         GEOMETRY(Geometry, 3857),
+    PRIMARY KEY (census_geo_id)
+);
+CREATE INDEX IF NOT EXISTS geo_type_index ON aggregate_us_demographics (geo_type);
+CREATE INDEX IF NOT EXISTS geom_index ON aggregate_us_demographics USING GIST (geom);
+
 -- Water-system-level data
 
 CREATE TABLE IF NOT EXISTS water_systems
 (
-    pws_id                    varchar(255) NOT NULL,
-    lead_connections_count    real,
-    pws_name                  varchar(255),
-    service_connections_count real,
-    population_served         real,
-    state_census_geo_id       varchar(255) references states (census_geo_id),
-    county_census_geo_id      varchar(255) references counties (census_geo_id),
-    geom                      GEOMETRY(Geometry, 4326),
+    pws_id                         varchar(255) NOT NULL,
+    pws_name                       varchar(255),
+    lead_connections_low_estimate  real,
```

Comment:
Same here I don't think we get it but we can verify with Jared

Reply:
Done.